client/debian/tests/run-hwctl-tests.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+set -xeu
+
+systemd-run --unit=mock-hw-server \
+            --property=NotifyAccess=all \
+            --property=Type=notify \
+            --slice=background.slice \
+            --quiet \
+            python3 "$(realpath "$(dirname "$0")")/mock_server.py"
+
+if ! systemctl is-active --quiet mock-hw-server.service; then
```

Comment:
Doesn't systemd wait for the service to start before returning? What's the point of this part?